core/src/main/scala/com/databricks/labs/remorph/discovery/SnowflakeTableDefinition.scala

```diff
@@ -0,0 +1,122 @@
+package com.databricks.labs.remorph.discovery
+
+import com.databricks.labs.remorph.parsers.intermediate.{DataType, StructField}
+import com.databricks.labs.remorph.parsers.snowflake.{SnowflakeLexer, SnowflakeParser}
+import org.antlr.v4.runtime.{CharStreams, CommonTokenStream}
+import com.databricks.labs.remorph.parsers.snowflake.SnowflakeTypeBuilder
+
+import java.sql.Connection
+import scala.collection.mutable.ListBuffer
+
+class SnowflakeTableDefinition(conn: Connection) {
```

Comment:
```suggestion
class SnowflakeTableDefinitions(conn: Connection) {
```
and rename a file as well